dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java

```diff
@@ -851,209 +853,219 @@ else if( fieldbcontentname.startsWith("date") ){
+					}
 
-				searchResult.put(fieldContentlet, fieldValue);
-			}
-			searchResult.put("inode", con.getInode());
-			searchResult.put("Identifier",con.getIdentifier());
-			searchResult.put("identifier", con.getIdentifier());
-			final Contentlet contentlet = con;
-			searchResult.put("__title__", getValueOrDefault (
-					() -> conAPI.getName(contentlet, currentUser, false), "Unknown"));
-			
-			String spanClass = (s.getStructureType() ==1)
-			        ? "contentIcon"
-			                :  (s.getStructureType() ==2)
-			                ? "gearIcon"
-			                        :  (s.getStructureType() ==3)
-		                        	? "formIcon"
-				                        :  (s.getStructureType() ==4)
-			                        	? "uknIcon " + UtilMethods.getFileExtension( ident.getURI()) + "Icon"
-			                        		:  (s.getStructureType() ==5)
-			                        		? "pageIcon"
-			                        				: "personaIcon";
-					                        
-			String typeStringToShow = s.getName() ;
-			searchResult.put("__type__", "<div class='typeCCol'><span class='" + spanClass +"'></span>&nbsp;" + typeStringToShow +"</div>");
-
-			String fieldValue = UtilMethods.dateToHTMLDate(con.getModDate()) + " " + UtilMethods.dateToHTMLTime(con.getModDate());
-
-			searchResult.put("modDate", fieldValue);
-			String user = "";
-			User contentEditor = null;
-			try {
-				contentEditor = APILocator.getUserAPI().loadUserById(con.getModUser(),APILocator.getUserAPI().getSystemUser(),false);
-			} catch (Exception e1) {
-				Logger.error(ContentletAjax.class,e1.getMessage() + " no such user.  did mod_user get deleted?");
-				Logger.debug(ContentletAjax.class,e1.getMessage(), e1);
-				contentEditor = new User();
-			}
+					searchResult.put(fieldContentlet, fieldValue);
+				}
+				
+				searchResult.put("inode", con.getInode());
+				searchResult.put("Identifier",con.getIdentifier());
+				searchResult.put("identifier", con.getIdentifier());
+				final Contentlet contentlet = con;
+				searchResult.put("__title__", conAPI.getName(contentlet, currentUser, false));
+
+				String spanClass = (s.getStructureType() ==1)
+						? "contentIcon"
+								:  (s.getStructureType() ==2)
+								? "gearIcon"
+										:  (s.getStructureType() ==3)
+										? "formIcon"
+											:  (s.getStructureType() ==4)
+											? "uknIcon " + UtilMethods.getFileExtension( ident.getURI()) + "Icon"
+												:  (s.getStructureType() ==5)
+												? "pageIcon"
+														: "personaIcon";
+
+				String typeStringToShow = s.getName() ;
+				searchResult.put("__type__", "<div class='typeCCol'><span class='" + spanClass +"'></span>&nbsp;" + typeStringToShow +"</div>");
+
+				String fieldValue = UtilMethods.dateToHTMLDate(con.getModDate()) + " " + UtilMethods.dateToHTMLTime(con.getModDate());
+
+				searchResult.put("modDate", fieldValue);
+				String user = "";
+				User contentEditor = null;
+				try {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)